### DISCUSSION TITLE
What is the purpose of OperationServers

### DISCUSSION THREAD

**Discussion by alexandrebrg** [ory/kratos#1972] [Q&A] [answered]
Hello, I'm currently working with Go SDK (previously was only working with TypeScript SDK) and I'm worried why there is a field named `OperationServers` in kratos configuration ([src](https://github.com/ory/kratos/blob/b56ff6e4ceeb7362b85e43171c900691af143358/internal/httpclient/configuration.go#L97)) ? 

Thank you!

**Comment by zepatrik** [comment, marked as answer]
The SDKs are generated using `openapi-generator`. In it's docs it says: https://github.com/OpenAPITools/openapi-generator/blob/c13067d100c43cfbedea205df37f365014cbdb2f/samples/client/petstore/go/go-petstore/README.md#urls-configuration-per-operation

**Reply by alexandrebrg** [reply]
Oh okay, didn't know about it thank you
